chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/listing/CertifiedProductEntity.java

```diff
@@ -345,14 +169,6 @@ public void setCreationDate(final Date creationDate) {
         this.creationDate = Util.getNewDate(creationDate);
     }
 
-    public Boolean getDeleted() {
-        return deleted;
-    }
-
-    public void setDeleted(final Boolean deleted) {
-        this.deleted = deleted;
-    }
-
     public Date getLastModifiedDate() {
```

Comment:
Same comment about possibly annotating the field? I'm looking at CuresUpdateEventEntity as an example.